ansible/configs/sap-integration/sample_vars.yml

```diff
@@ -0,0 +1,119 @@
+cloud_provider: osp
+env_type: sap-integration
+output_dir: /output
+
+
+guid: sapi
+
+repo_method: satellite
+satellite_org: MY_SATELLITE_ORG
+satellite_activationkey: MY_ACTIVATION_KEY
+satellite_url: MY_SATELLITE_URL
+use_content_view: true
+
+ocp4_pull_secret: 'MY OCP PULL SECRET'
+
+# Authenication credentials for OpenStack in order to create the things.
+# These should be included with your secrets, but are listed here for reference
+# osp_auth_url:
+# osp_auth_username:
+# osp_auth_password:
+# osp_auth_cloud:
+# osp_auth_project_domain: #usually set to "default"
+# osp_auth_user_domain: #usually set to "default"
+# osp_project_name: 
+# osp_project_id:
+# osp_project_create: false (this must be false when tested in sandbox)
+
+# These should be included with your secrets, but are listed here for reference
+osp_cluster_dns_server: FROMSECRET
+osp_cluster_dns_zone: FROMSECRET
+ddns_key_name: FROMSECRET
+ddns_key_secret: FROMSECRET
+
+osp_use_swift: false
+software_to_deploy: openshift4
+
+# -------------------------------------------------------------------
+# Infra Workload Variables
+# -------------------------------------------------------------------
+
+# Authentication (HT Passwd)
+ocp4_workload_authentication_idm_type: htpasswd
+ocp4_workload_authentication_admin_user: admin
+# When no password specified it is generated
+ocp4_workload_authentication_htpasswd_admin_password: PASSWORD_FOR_ADMIN
+ocp4_workload_authentication_htpasswd_user_base: integration
+# When no password specified it is generated
+# ocp4_workload_authentication_htpasswd_user_password: ocp_student
+
+# Create 5 student users
+ocp4_workload_authentication_htpasswd_user_count: 1
+
+# Remove the standard kubeadmin user
+ocp4_workload_authentication_remove_kubeadmin: false
```

Comment:
Do you want to keep kubeadmin when you already have an admin user with the identity provider?

Reply:
Ok, this just reminded me of one thing. When is this removed? I mean, .kubeconfig for kubeadmin is used to deploy the workloads. If this is removed after that I'm fine removing it.

Reply:
ok, you are right, `system:admin` credentials are used. Removed.